command/meta.go

```diff
@@ -25,30 +29,35 @@ type nameHelpCommand interface {
 	Help() string
 }
 
-func (m *meta) defaultFlagSet(c nameHelpCommand, args []string) *flag.FlagSet {
-	flags := flag.NewFlagSet(c.Name(), flag.ContinueOnError)
+func (m *meta) defaultFlagSet(name string) *flag.FlagSet {
+	flags := flag.NewFlagSet(name, flag.ContinueOnError)
+	flags.IntVar(&m.port, "port", config.DefaultPort, "The port to use for the Consul Terraform Sync API server")
 	flags.SetOutput(ioutil.Discard)
-	flags.Usage = func() {
-		m.UI.Error(fmt.Sprintf("Error: unsupported arguments in flags '%s'",
-			strings.Join(args, ", ")))
-		m.UI.Output(fmt.Sprintf("Please see --help information below for "+
-			"supported options:\n\n%s", c.Help()))
```

Comment:
This part outputs an error when the user includes unknown flags e.g. `consul-terraform-sync disable task  -stuff my_task`. I think it would be helpful to include an error message for this edge case. Do you have something else in mind?

Reply:
Oops I had missed this, I was tweaking this and moving it around and it ended up being removed. Completely unintentional

Reply:
I had difficulty with how `meta` and the commands were interacting with each other. The command was embedding meta but meta required the command within functions.

Made some changes to tease the circle dependency apart.

Reply:
Oh yea, command calling meta calling command was messy on my part. Thanks for catching and fixing this!